README.md

```diff
@@ -35,7 +35,17 @@ encrypted with which key, making it easy to age out keys when needed
 Keys are managed through a keyring--a short JSON document containing
 information about your encryption keys. The keyring must be a JSON
 array of objects with the fields `id`, `created_at`, and `value`. A
-keyring must have at least one key.
+keyring must have at least one key. For example:
+
+```json
+[
+  {
+    "id": "1380e471-038e-459a-801d-10e7988ee6a3",
```

Comment:
What do you think about providing a numeric id here instead? I've come to realize encouraging uuids for key ids is kind of silly: their advantages are, for all practical purposes, moot in this context. Plus with numeric ids, the `created_at` can be considered superfluous. I'd like to overhaul keys to use numeric ids, and only support uuid plus `created_at` as a fallback, and maybe we can start with using a numeric id in the example?